### DISCUSSION TITLE
参考文献作者全部大写

### DISCUSSION THREAD

**Discussion by LiY321** [sjtug/SJTUThesis#728] [提问]
参考文献里面作者的名字全部都大写了，怎么改成正常的首字母大写呢
![1636984843(1)](https://user-images.githubusercontent.com/65121771/141794645-cc0df3a2-9930-42e5-9e30-c9aed7dfe176.png)

**Comment by skyzh** [comment]
目前的样式是符合 gb7714-2015 标准的。如果你一定要修改，可以在 `setup.tex` 中将 biblatex 的这一行改成

```
\usepackage[backend=biber,style=gb7714-2015,gbnamefmt=lowercase]{biblatex}
```

`gbnamefmt=lowercase` 可以将姓名原样引入。
